WeatherMood/Extension/UIView+.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIView+.swift
+//  WeatherMood
+//
+//  Created by 이지원 on 2021/07/27.
+//
+
+import UIKit
+
+extension UIView {
+    static var reuseIdentifier: String {
+        String(describing: Self.self)
+    }
+
+    /// 하위 뷰 목록 끝에 뷰(들)을 추가합니다.
+    /// - Parameter views: 추가할 뷰(들)입니다. 추가된 후 이 뷰(들)은 다른 모든 하위 뷰 위에 나타납니다.
+    func addSubviews(_ views: UIView...) {
+        for view in views {
+            self.addSubview(view)
```

Comment:
클로저나 self가 반드시 필요한경우에만 `self`를 붙이는건 어떨까요?  이유는 저번에도 말씀드렸어서 아실꺼라 생각합니당!
```suggestion
            addSubview(view)
```

Reply:
그렇긴 한데, `UIView` 익스텐션이라 `self`를 안 붙이면 뭔가 모호한 느낌이 들어서 썼습니다 ;(

Reply:
넵 알겠습니당~